pyproject.toml

```diff
@@ -8,17 +8,12 @@ dynamic = ["version"]
 description = "Smart object creation facility for Django."
 readme = "README.md"
 requires-python = ">=3.8"
-license = {text = "Apache License 2.0"}
+license = { text = "Apache License 2.0" }
```

Comment:
Are these changes coming from some linter?

Can we take this (and other similar cosmetic changes down there) out of this PR (as it does not belong to adding a new Python version support) and discuss it separately? 
Don't get me wrong, I am happy to implement it, but it needs to be added separately.

Reply:
It's from my autoformat code editor 🤦 
I will revert.

Reply:
@jairhenrique do you mind sharing, which one is it? I'd be curious to try and maybe even add it to the CI/dev-tools here.